My Simple App/Services/Event.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+/// One to many communication of a value's updates.
+///
+/// # Example
+/// ```
+/// class KeyboardListner {
+///   let keyboardHeight: Event<CGFloat>
+///
+///   ...
+/// }
+///
+/// class MyViewController {
+///   let keyboard = KeyboardListner()
+///
+///   init() {
+///     self.keyboard.keyboardHeight.add(subscriber: self) { [weak self] newHeight in
+///       self?.updateUIToNewKeyboardHeight(newHeight)
+///     }
+///   }
+/// }
+/// ```
+class Event<Value> {
```

Comment:
Since Combine hasn't been used in our codebase and it's not something SUPER familiar with most iOS devs, I decided not to use Combine but I do agree this would be a great application for Combine